### ISSUE TITLE
windows exe - text delete

### ISSUE THREAD

**Issue by joejarlett** [michelolvera/vs-ghostwriter#28]
I couldn't use backspace or the del button in the text editor
windows 10 vs-ghostwriter version 1.8.1

**Comment by miker** [comment]
Are you sure you weren't in "Hemingway Mode"? That mode disables the backspace and delete keys (ie typewriter mode) to force you to just write instead of potentially wasting time editing.

**Comment by PrSunflower** [comment]
I agree. @joejarlett please check that Hemingway Mode is disabled, this would be too much of a major bug.
